thrust/thrust/random/detail/normal_distribution.inl

```diff
@@ -33,7 +33,7 @@
 #include <cstdint>
 
 // for floating point infinity
-#if THRUST_DEVICE_COMPILER == THRUST_DEVICE_COMPILER_NVCC
+#ifdef _CCCL_CUDA_COMPILER
```

Comment:
Nitpick: This should just be `#include <cuda/std/limits>`. Feel free to open an issue